circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Looks like [this](https://discuss.circleci.com/t/announcing-xcode-8-3-support-and-opt-in-upgrades-for-macos/11528/21) fixed.